packages/aws-cdk-lib/aws-s3-deployment/README.md

```diff
@@ -375,10 +375,12 @@ import * as lambda from 'aws-cdk-lib/aws-lambda';
 
 declare const myLambdaFunction: lambda.Function;
 declare const destinationBucket: s3.Bucket;
+declare const destinationKey: string;
```

Comment:
Can we add a short description on top to explain the `destinationKey` prop?

Reply:
Added comment in the `README.md` file, but the source code itself has comments on the property as well.